File: demo-vk-rest-scraping.java/src/main/java/demo/model/School.java
package demo.model;

public class School {

    private final int id;
    private final String title;

    private School(int id, String title) {
        this.id = id;
        this.title = title;
    }

    public School(ConstructorMediator cm) {
        this(cm.getId(), cm.getTitle());
    }

    public int getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    @Override
    public String toString() {
        return "School{" +
                "id='" + id + '\'' +
                ", title='" + title + '\'' +
                '}';
    }
}

File: demo-vk-rest-scraping.kotlin/src/main/kotlin/demo/model/School.kt
package demo.model

class School private constructor(val id: Int, val title: String) {

    constructor(cm: ConstructorMediator) : this(cm.id, cm.title) {
    }

    override fun toString(): String {
        return "School{" +
                "id='" + id + '\'' +
                ", title='" + title + '\'' +
                '}'
    }
}
